core/templates/dev/head/pages/about/about.html

```diff
@@ -480,6 +479,15 @@
       });
     });
   </script>
+  <style type="text/css">
+    .button_style {
```

Comment:
This is not a great name for a class, and we don't use underscores in class names. in the future, please look at the codebase and make sure your code conforms to the existing conventions before submitting.

Perhaps you could call this .oppia-about-foundation-tab-button instead of .button_style.